src/scss/grommet-core/_objects.menu.scss

```diff
@@ -3,6 +3,19 @@
 .menu {
   position: relative;
   @include inuit-font-size($menu-font-size, inherit);
+  
+  .header & {
+    
+    &--controlled,
+    &__control {
+      height: 100%;
+    }
+    
+    &__control {
+      line-height: double($inuit-base-spacing-unit);
```

Comment:
I think this will be dependent on the header hight. Would it work to use inherit to get the same line-height as the containing header?